packages/wrangler/src/pages.tsx

```diff
@@ -12,6 +12,7 @@ import { generateConfigFromFileTree } from "../pages/functions/filepath-routing"
 import { writeRoutesModule } from "../pages/functions/routes";
 import openInBrowser from "./open-in-browser";
 import { toUrlPath } from "./paths";
+import { pagesBetaWarning } from ".";
```

Comment:
This will cause a circular dependency, which we should avoid.
Please can you move the `pagesBetaWarning` export into this file and import it into the top level file, instead.

Reply:
Nice catch, I should've noticed that.